rex/resource.py

```diff
@@ -985,30 +1004,38 @@ class MultiH5:
     Class to handle multiple h5 file Resources
     """
 
-    def __init__(self, h5_dir, prefix='', suffix='.h5'):
+    def __init__(self, h5_files, check_files=False):
         """
         Parameters
         ----------
-        h5_dir : str
+        h5_files : str
```

Comment:
Definitely update this doc string

Reply:
dope...

Reply:
This still has `h5_files : str` even though the description is now accurate :)